build/requirements.txt

```diff
@@ -1,2 +1,2 @@
-jaxlib==0.6.0
-jax==0.6.0
+jaxlib==0.7.0
```

Comment:
This is probably fine for right now, but if we could use JAX 0.7.1 and delete the namespace patch (or at least take it out of the Docker build) in a follow-up PR, that'd be awesome. 0.7.1 includes Matt's change that gets jaxlib to load our plugin.